docs/pages/icon.js

```diff
@@ -1,133 +1,203 @@
+          title="Legibility and touch area"
+          description="Ensure that icons use a contrast ratio of 4.5:1 between icon color and background color."
+        />
+      </MainSection>
+      <MainSection name="Localization" description="Be sure to localize `accessibilityLabel`." />
+      <MainSection name="Variants">
+        <MainSection.Subsection title="Primary-color combinations">
+          <CombinationNew color={['gray', 'darkGray', 'red']}>
+            {({ color }) => <Icon icon="heart" accessibilityLabel="" color={color} />}
+          </CombinationNew>
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Size"
+          description={`
+These are the guidelines for icon sizes (in px):
+1. \`12\`
+Used sparingly in tight spaces. When possible, use a text label next to the icon, as it can be hard to see for visually impaired people. |
+2. \`16\`
+Used often any time an icon is needed. Default icon size.
+3. \`24\`
+Used frequently any time an icon is needed.
+4. \`32\`
+Used occasionally, on more dense UI.
+5. \`32+\`
+Should be used sparingly and only in places where the UI is very dense and a larger icon is required.
 `}
-  />,
-);
-
-card(
-  <Combination id="iconCombinations" name="Icon Combinations" icon={icons}>
-    {(props) => <Icon color="darkGray" accessibilityLabel="" size={32} {...props} />}
-  </Combination>,
-);
+        />
+      </MainSection>
+      <MainSection name="Writing">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description={`
+- Use a descriptive label to describe the Icon
+- Be succinct. Exclude unnecessary words.
+- Be informative and accurate
+- Write in the active voice
+- Avoid technical jargon
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description={`
+- Use the words "image" or "icon" in the description label; instead, use words that indicate the purpose of the icon.
+`}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+      <MainSection name="Related">
+        <MainSection.Subsection
+          description={`
+**[IconButton](/IconButton)**
+Use IconButton when only an icon is needed to represent an action instead of text.
 
-card(
-  <Combination
-    id="sizeColorCombinations"
-    name="Size & Color Combinations"
-    size={[16, 24, 32]}
-    color={['gray', 'darkGray', 'red']}
-  >
-    {(props, i) => <Icon key={i} icon="heart" accessibilityLabel="" {...props} />}
-  </Combination>,
-);
+**[Button](/Button)**
+Use Button to allow users to take an action.
+    `}
+        />
+      </MainSection>
+    </Page>
+  );
+}
 
-export default function IconPage(): Node {
-  return <CardPage cards={cards} page="Icon" />;
+export async function getStaticProps(): Promise<{| props: {| generatedDocGen: DocGen |} |}> {
+  return {
+    props: { generatedDocGen: await docgen('Icon') },
```

Comment:
@AlbertCarreras a proposal to make the `icon` type on `Icon` work: https://github.com/christianvuerings/gestalt/commit/71faa1d62828c8203edb57ab270c3fba4196d6d3

Reply:
DONE